docs/source/tutorials/GEDI/SearchGEDI.ipynb

```diff
@@ -0,0 +1,482 @@
QQOLC4O4GcVKHCgqTiAhs32nW2KMgB2+I4uyYSGncBMgQIH\nAgfmc2CN1bdzM0pn6n6lv/78p2hIJdqvmPtWittH/+W7uo/GvKoa1uryWtciLxFh28k2MviwThg/\nRiL09nJrrLOtG/3BMwpEq6okym9eC9e6ZWsFrmhp0cADRrlvnpjRtxBrnRpBsi24JnMLe6YWyfcG\na1LnP1b4FjgQOBA4EDgQOBA4EDgQOBA4EDgQOBA4sCRygFhDLHKVl0+V7ch+dtNK56pWM180rL+P\nx/xX/FXZP0Z8TBOAU4ClAMp55ZVu9vifJbDYau67735xa62/lyubO1nMdyV+QqtOrnT6TDdbYhu0\nEMA6ccIfrotsDfbzL+PcDHFJaNu+o5sxbYq4wMq2M6KVxcy4S9deAaQuiR0o1DlwIHAgcCBwIHAg\ncCBwIHAgcCBwIHCgMTnw4ehnVMsJABU8qUAULWet7JmaVysnhGprJaiSnNM0eWLGK2lLSyUqsARG\n+uD9x93aEmBpuuydWlYmgfQK81z7vArXoXN7iU9Q4EpnTHY9u4v/ac1UV1DcyvVedbC4qK7s3n37\nYS0XLStAeMCgYQGkNmbDhrwCBwIHAgcCBwIHAgcCBwIHAgcCBwIHlkgOAEhxCsUcF6AqH1GbqvZU\ngSrn8BqV/3qp7iH1FkG1NQJg3x71oN7O9jPVYu5bJtvPFEpEX75zUyLKbwv1ZS2WwGsTJlW4lVfd\nsS6nxGHi+EkBpC7AkfAjcCBwIHAgcCBwIHAgcCBwIHAgcCBw4G/MAbSZCkcVseI+mgCvoucUrarE\nQAKsyqdG/Epra6slwncBVxJAVNJWy/maagmCKd/zCZokKJa0CmzrvhMcacwnb0rE//bu+6+/EfBa\nkMhHzpdLAM3gk/o37oDh0QMHAgcCBwIHAgcCBwIHAgcCBwIHAgfgQK0ASQgwyXdMfgGYRgpeUa6K\njypWv2IDnDjKj3llFQpcW8reqgBT8G2RgNdWrYoV8LJVDSfZ1gxAy/63bDdTXl4mpwG7LeVsoauS\n/VXRviaMi63kcAwcCBwIHAgcCBxYRBx44oknJDDD5EVUWsOLKS0tdQ8++KBmVCERCe+44w7xwylt\neMY55vDWW2+5//3vf27WrFkZ7/zqq68c6RtKbOzOxuzRD4JFY1CmelI+2xwsDvr6668bhYeLuu60\nzUMPPaR7pS7qsq28H374wb3++uv2c4HjG2+8oXsFL3Cy7kdDxtfiHKcNqXccHzhHv4+OO/vNdebQ\nRx99lK+NRpT53Xffxc4xVRLl1cq3o41Nf56wc9FKpRvLH3/8saNfBFp8HCCwUbVsE8NHYvDKR3Sn\nAFcBpHl8RI3Kx8l0zHkFqpKuvLxKLrZwBeKXSrTevIJC3RN1ngRimjx1hkQMLpU8K8Xst8AVyaeg\nsIUrFM0pCHgeW9VIRm3atta9VwGzgNygSV18/SCUHDgQOBA48LfmwH/+8x/Xq1cv102i/DUWzZgx\nw91///3uwAMPdI29AfzYsWPdNddc4/bYYw/3xx9/uFtvvdUNHDhQohmu21jVz5jPk08+6f7v//7P\nbbzxxm7bbbddKP0rr7wiq9at3AYbbKDXXnvtNffNN980eDuSe++91910000Llcc+k/XZxiPXep5z\nzjlu6tSpujfnQpVo4hOvvvpqo/Cwiau5UPaA1Kuvvlq3iunUqdNC1xfFibfffltB6rBhwxYqjr4z\nfPhwt8IKKyx0rSHja3GO02i9o/18oQfN4sQhhxySEszfeOONqo2inXfdddcsckufBND5z3/+073z\nzjsKjkm97LLLOuZq2xN30003TV6z3FZZZRVHXaLzBP1utdVWc0cffbTr2bOnJifN448/7h577DG7\nPXl85JFH3Pjx413YoivJkkX+Zb0NdhTIKRi0tlz2L31RNZwAVKL+Yt6bL5F32SJG7H0VxKrPqtrw\nJqqKSS97oeYBYGUOYlECTFsuwLNGIgJXyQdfVSL4ssZZLNvbtBR/VdJOnTpNwG6lll8oYDeA1EXe\n/KHAwIHAgcCBwIGm4gAg9Z577lEg2dgg1a9zSUmJGzVqlPjPFPqnm/z76NGj3frrr+/+/e9/x5b1\n/PPPu/bt2ydBamyiep7s0qWLu+WWWxa4u3v37gv8zvZHrvX8+eefY0F5tuWFdEsWBxprfDVWPtly\nL1perv08rpzLL7/coaGF9ttvP/1stdVW+psx+cknn+j3xvgDQH3//fd1IWz11VdXcHzRRRe5gw8+\n2D399NMKUijnpJNOWmCOYWHMyOYJ6oyFxJ133ukOOuggd/fddzuupaOLL744AYrSJQrXmpQDFZXV\nbvbs2a5WgKZqUAVgYppbgF8pmlX5rcBUQCV7oHIN0nNyBGzqeUBq3TXuwf/UVSXOAGTJL18i+aIx\nxb8VE+A5s+c6sRRWAMu+rAGkKmvDn8CBwIHAgcCBpubAAw88oCvorNYfeuihCxTHaisr8ZgGtmnT\nxu2yyy5uxx0T0f4wbZ02bZpqLz/77DOHNhHBDM0epm5oM0877TQJd1+mwhMZk/9uu+3m9tlnHwew\nu+GGG9zEiRNd79693amnnuqWW245fRGjpSANZpEQ5fKBELKuvPJK1SogXJlgyDV8aUaMGOGuuuoq\nCavf1qXLh/QA52eeeUbzpG7w4swzz3QDBgzg8gJEOrQMU6ZMcWuttZY77rjjdN86gCmaKYg6Axgp\n2whtx0cffeTyZTN0TPUQDiH4zb1oR5Zaail3xhlnqHaEa1988YW7/fbbHWaZaDwoK5VmGyHCtCHc\n61MqHlMfnp024rkQgjHny7WeaKhMO4ywS16090YbbeSOPfbYWK25PRu86N+/vzv++OPd8ssvr9XG\nbBttDsLYqquuqjxp166d1itaX27AdJG+gFYV7T9C+9prr6150XfpX5hVI6ihaaKN+
```

Comment:
1. Remove leading `#### `
2. Change "maap" to "MAAP"
3. Change "getLocalPath" to "`getData`" (because getLocalPath is deprecated)
4. Add period at the end